rs/nns/governance/src/governance/tests/mod.rs

```diff
@@ -133,6 +133,84 @@ mod settle_neurons_fund_participation_request_tests {
     }
 } // end mod settle_neurons_fund_participation_request_tests
 
+#[cfg(feature = "test")]
```

Comment:
Is this needed? Is there any new features not enabled, so that this needs to be behind `feature = "test"`?

Just wanted to prevent the case where you actually intended to have it under `#[cfg(test)]`, which is already the case.